plugins/rplanners/cubicretimer.cpp

```diff
@@ -254,18 +254,18 @@ class CubicTrajectoryRetimer : public TrajectoryRetimer
             _v0vel[i] = *(itdataprev+info->gvel.offset+i);
             _v1vel[i] = *(itdata+info->gvel.offset+i);
         }
-        dReal deltatime = *(itdata+_timeoffset);
```

Comment:
@Puttichai can you check this out? Was this intentional to have dead code like this? or did you not finish something?

Reply:
This line was last changed 10 years ago [here](https://github.com/rdiankov/openrave/commit/658a6e1b09b8c17d1005a0dce8231386aa6a0c9d#diff-379b23408e41198919e1c9a87048dec7bc2da2f13c06ee412cf13f3daa3052b8R235) if it helps